skaffold.yaml

```diff
@@ -122,13 +114,6 @@ profiles:
           <<: *docker_args
           dockerfile: deployments/Dockerfiles/documentation/Dockerfile
 
-      - image: valory/oar-oracle
```

Comment:
Make sure we have a skaffold.yaml in the apy and price repos

Reply:
I will open PRs for that. Also, take a look at https://github.com/valory-xyz/open-autonomy/pull/1441#discussion_r998185966 re duplication